src/components/Jam.jsx

```diff
@@ -157,7 +158,7 @@ export default function Jam() {
     const body = { destination_addresses: destinations }
 
     if (process.env.NODE_ENV === 'development' && useInsecureTestingSettings && !deactivateTestingToggle) {
-      body.tumbler_options = {
+      body.scheduler_options = {
```

Comment:
Are you panning to update [your server branch](https://github.com/dnlggr/joinmarket-clientserver/tree/tumbler-rpc-options) or the [the PR](https://github.com/JoinMarket-Org/joinmarket-clientserver/blob/dd8e08ffb6ebdfe9f73307575079cd2123e51e4d/jmclient/jmclient/wallet_rpc.py#L1119) as well? Both still refer to `tumbler_options`.. would need to remember that till they are in sync again..

Reply:
Oh yeah that needs to be fixed of course!

Reply:
Fixed. Good catch! 👏